src/utils.hack

```diff
@@ -34,7 +34,8 @@ function print_type(mixed $value): string {
 }
 
 function is_iterable(mixed $value): bool {
-  return \is_array($value) || (\is_object($value) && ($value instanceof Traversable));
```

Comment:
Warning: This change combines the replacement of `instanceof` with a hackfmt move. I did not notice at the time.